src/Storage/Storage.Management/help/New-AzDataLakeGen2Item.md

```diff
@@ -0,0 +1,348 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/az.storage/new-azdatalakegen2item
+schema: 2.0.0
+---
+
+# New-AzDataLakeGen2Item
+
+## SYNOPSIS
+Create a file or folder in a container.
+
+## SYNTAX
+
+### File (Default)
+```
+New-AzDataLakeGen2Item [-Container] <String> [-Path] <String> -Source <String> [-Umask <String>]
+ [-Permission <String>] [-Property <Hashtable>] [-Metadata <Hashtable>] [-Force] [-AsJob]
+ [-Context <IStorageContext>] [-ServerTimeoutPerRequest <Int32>] [-ClientTimeoutPerRequest <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-ConcurrentTaskCount <Int32>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### Folder
+```
+New-AzDataLakeGen2Item [-Container] <String> [-Path] <String> [-Folder] [-Umask <String>]
+ [-Permission <String>] [-Property <Hashtable>] [-Metadata <Hashtable>] [-Force] [-AsJob]
+ [-Context <IStorageContext>] [-ServerTimeoutPerRequest <Int32>] [-ClientTimeoutPerRequest <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-ConcurrentTaskCount <Int32>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzDataLakeGen2Item** cmdlet creates a file or folder in a container in an Azure storage account.
+This cmdlet only works if Hierarchical Namespace is enabled for the Storage account.
```

Comment:
From this cmdlet help itself, there is no reference about Hierarchical Namespace configuration. How we are going to cover this fundamental context for PSH ADLS Gen2 customers, in cmdlet help or a blog for PSH cmdlet?

Reply:
I think it should be covered by the doc.
Anyway, I  can also add the way to enabled it in help.